Prevent pause menu engine halt

helenclarko opened the pull request:
If the toggle engine key is set to mouse wheel, this will prevent the engine stop/start when in the pause menu.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
- Does your code fit the style guidelines? Yes
- Does your PR fit the contribution guidelines? Yes

helenclarko committed:
Prevent pause menu engine halt

If the toggle engine key is set to mouse wheel, this will prevent the engine stop/start when in the pause menu.

helenclarko committed:
Update client.lua

jnccloud reviewed (changes requested):
Please update this to use the locale system.

helenclarko committed:
Added language notification back into code

Previous commit had wiped out language, this commit adds the language options back in.

helenclarko commented:
Sorry about that, corrected in latest commit.

GhzGarage commented:
@loljoshie

Z3rio commented:
No longer valid PR (because of it being outdated)
